cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/utils/NGSICharsets.java

```diff
@@ -162,6 +166,51 @@ public static String encodeCKAN(String in) {
         
         return out;
     } // encodeCKAN
+
+    /**
+     * Encodes a string for SlicingDice. Only lowercase alphanumerics and - are allowed.
+     * @param in
+     * @return The encoded string
+     */
+    public static String encodeSlicingDice(String in) {
```

Comment:
97, 119, 121, 122, 48, 57...

They may appera as magic number for people not very familiar with encoding. Maybe it would be a good idea to include a little explanation on how this function works in the JavaDoc preamble (explaining the difreent ranges/exceptions).